taxi/forms.py

```diff
@@ -0,0 +1,57 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm, get_user_model
+
+from taxi.models import Car, Driver
+
+
+class DriverLicenseValidateMixin:
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "License number must be 8 characters long"
+            )
+        if (
+            license_number[:3].upper() != license_number[:3]
+            or license_number[:3].isalpha() is False
+        ):
+            raise forms.ValidationError(
+                "The last 5 characters of the license number must be digits"

```

Comment:
The error message here is misleading. It should reflect the actual validation logic being performed. Consider changing it to something like 'The first 3 characters of the license number must be uppercase letters.'